pkg/bloomcompactor/bloomcompactor.go

```diff
@@ -564,11 +576,10 @@ func (c *Compactor) runCompact(ctx context.Context, logger log.Logger, job Job,
 	if err := ctx.Err(); err != nil {
 		return err
 	}
-
 	metaSearchParams := bloomshipper.MetaSearchParams{
 		TenantID:       job.tenantID,
-		MinFingerprint: uint64(job.seriesFP),
-		MaxFingerprint: uint64(job.seriesFP),
+		MinFingerprint: uint64(job.minFp),
```

Comment:
these values are not initialised at this moment yet. you need to call `computeBounds` at the beginning.

Reply:
good catch, addressed right after job creation.